b2share/modules/deposit/views.py

```diff
@@ -136,8 +140,14 @@ def records_rest_url_rules(endpoint, list_route=None, item_route=None,
         links_factory=links_factory,
         default_media_type=default_media_type)
 
+    from b2share.modules.records.views import RecordsVersionsResource
+    versions_view = RecordsVersionsResource.as_view(
+        RecordsVersionsResource.view_name.format(endpoint),
+        resolver=resolver)
```

Comment:
This will be removed. We only keep `/api/records/<ID>/versions`. We don't want a duplicate `/api/records/<ID>/draft/versions`